frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/cell/BranchOrCommitCellRendererComponent.java

```diff
@@ -72,7 +74,8 @@ public BranchOrCommitCellRendererComponent(
       int row,
       int column) {
 
-    this.graphTable = table;
+    assert table instanceof GitMacheteGraphTable : "Table variable is not instance of ${GitMacheteGraphTable.class.getSimpleName()}";
```

Comment:
Hmm that's not the cleanest approach... what about baking `ongoingRepositoryOperation` into `RepositoryGraph`, `BranchItem` or maybe even `GitMacheteBranch` instead?

Reply:
Yeah, possibly adding `Option<OngoingRepositoryOperation> getOngoingRepositoryOperation()` to `IGitMacheteBranch` would be pretty clear... but then likely also `isCurrentBranch` would be needed to be added to `IGitMacheteBranch` as well (and possibly `IGitMacheteRepositorySnapshot#getCurrentBranchIfManaged` could be removed)

Reply:
But `ongoingRepositoryOperation` is something common for whole repository, so baking it into any structure representing branch is IMO not adequate.

Reply:
Hmm it could theoretically be a property of the current branch... although I do admit that it's indeed more a property of a repository :/ but still, I'm against doing `instanceof` here... how about just changing `graphTable` field type to `GitMacheteGraphTable`?

Reply:
Also, is current impl working with `DemoGraphTable`? :thinking:

Reply:
As agreed via Slack I'll try to implement this in way with common interface